### DISCUSSION TITLE
SX1262 RX Gain

### DISCUSSION THREAD

**Discussion by mfvmx** [jgromes/RadioLib#370] [Q&A]
The datasheet show a command for changing the RX gain is this in the current library or am I over looking it?
![image](https://user-images.githubusercontent.com/5554014/134192052-dbf14df0-40a5-4864-ab31-e579183da7cb.png)

**Comment by mfvmx** [comment]
I see the register is defined but I don't see it used anywhere.
https://github.com/jgromes/RadioLib/blob/83d77e1ac5a2a0057c76774656a3f63d983a82f2/src/modules/SX126x/SX126x.h#L95

**Comment by jgromes** [comment]
This is currently not implemented for a very simple reason - what exactly it means to "consume more power to improve the sensitivity" seems to be undocumented. Is it an improvement of 10% or 0.1%? And is it at the expense of 5% of extra power consumption or 50%?

**Reply by mfvmx** [reply]
Well that's what I hoped to find out. I have found it used in another library and can update you with the results.
